client/state/happychat/selectors/is-happychat-available.js

```diff
@@ -8,12 +8,17 @@ import { get } from 'lodash';
  * Internal dependencies
  */
 import isHappychatClientConnected from 'state/happychat/selectors/is-happychat-client-connected';
+import isHappychatUserEligible from 'state/happychat/selectors/is-happychat-user-eligible';
 
 /**
  * Returns true if Happychat client is connected and server is available to take new chats
  * @param {Object} state - global redux state
  * @return {Boolean} Whether new chats can be taken
  */
 export default function( state ) {
```

Comment:
There are use cases where we'll need happychat availability and user eligibility separately, so we'll need a new selector `isUserEligibleForHappychat` and `is-happychat-available` as it was.

Reply:
Just saw the `is-happychat-user-eligible` selector. We'll need to revert the changes to `is-happychat-available`, though.